lib/library.rb

```diff
@@ -0,0 +1,69 @@
+require 'yaml'
+require 'date'
+require 'pry'
+require './lib/person.rb'
+
+class Library
+
+attr_accessor :items
+
+def initialize
+  @items = YAML.load_file('./lib/data.yml')
+end
+
+def check_out(title, person)
+ case
+ when
+   if_not_available?(title, person) then 'Book is not available'
+ else
+   perform_checkout(title, person)
+ end
+end
+
+def see_list
+  items.each {|x| puts x }
+end
```

Comment:
Hard no here. `puts` is used for terminal commands only. This method doesn't do anything for your code.